doc/changelog_user_guide.md

```diff
@@ -0,0 +1,396 @@
+# Changelog User Guide
+
+`dotnet-releaser` allows to generate changelogs that will be pushed as release notes. It **supports 2 modes**:
+
+- **Automatic**
+
+  This is the default mode that will generate automatically pretty changelogs directly from your pull-requests and commit changes, fully customized with categories, auto-labelers, contributors/labels filtering and templates. If you are familiar with solutions like [Release Drafter](https://github.com/release-drafter/release-drafter), `dotnet-releaser` provides similar integrated features and goes beyond.
+
+- **Manual**
+
+  This mode is more interesting if you are writing your changelog manually in a file and want to transfer this automatically to your release notes.
+
+Table of Content
+
+- [1. Automatic changelogs](#1-automatic-changelogs)
+  - [1.1 Overview](#11-overview)
+  - [1.2 Properties](#12-properties)
+  - [1.3 Categories](#13-categories)
+  - [1.4 Auto-labelers](#14-auto-labelers)
+  - [1.3 Templates](#13-templates)
+    - [1.3.1 Commit template](#131-commit-template)
+    - [1.3.2 Pull-Request template](#132-pull-request-template)
+    - [1.3.3 Body template](#133-body-template)
+    - [1.3.4 Extra template properties](#134-extra-template-properties)
+- [2. Manual changelogs](#2-manual-changelogs)
+- [3. Disabling changelogs](#3-disabling-changelogs)
+  
+
+## 1. Automatic changelogs
+
+### 1.1 Overview
+
+By default, this feature is enabled and you don't have to configure anything to get the benefit of it. Though, the automatic changelogs is highly customizable with many configuration knobs, from defining auto-labelers or the [Scriban](https://github.com/scriban/scriban) templates used to generate the changelog. The following sections are describing all the possibilities of configuration. 
+
+The image below is an example of an auto-generated changelog from pull-requests and commits on the project [Tomlyn](https://github.com/xoofx/Tomlyn/releases).
+
+![changelog](changelog.png)
+
+### 1.2 Properties
+
+| `[changelog]`                         | Type       | Description                |
+|---------------------------------------|------------|----------------------------|
+| `publish`                             | `bool`     | Enable to disable changelog publish. Default is `true`.
```

Comment:
The description of this flag is wrong. (Enable to **disable** changelog publish)
Shouldn't it be something like: Whether changelogs should be published.